### DISCUSSION TITLE
Facing an Error, Problem Installing Ansible in Mizoram State SDC

### DISCUSSION THREAD

**Discussion by lokeshshivamurthy** [Sunbird-cQube/community#853] [Q&A]
The following issue is Faced by Mizoram State, while setting up cQube-v5.0.3 in their SDC(State Data Centre). 
Note: SDC do not have any cloud servers. The following installation was done in their Virtual Machine.

1. First Installed Ubuntu-22.04. (link: As provided in your website link) ( https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview).
2. After Installing ubuntu desktop, Mizoram State started doing the installation.
3. Cloning from :  git clone https://github.com/Sunbird-cQube/cqube-devops.git
4. Release branch: release-v5.0.3
5. Access type: VSK
6. Mizoram Code : MZ
7. Storage type: local

After providing all details in installation, The installation starts and then we are Facing Error " Error there is a problem Installing Ansible"
Attaching few screenshots for errors.
![Ubuntu Version](https://github.com/Sunbird-cQube/community/assets/134672230/f601800a-9015-4158-a7ce-5f86d9c8b68b)
![Ansible Error](https://github.com/Sunbird-cQube/community/assets/134672230/500e669b-6483-449f-963b-80b263910a4a)

Any suggestions  ??

**Comment by RNarmatha** [comment]
Hi @lokeshshivamurthy , 

This is related to ansible installation issue. 

Please run the below command and share the screenshot
ansible --version

**Comment by lokeshshivamurthy** [comment]
![ansible --version](https://github.com/Sunbird-cQube/community/assets/134672230/fa502968-967f-42cd-9d73-d797118eaf59)

Output of command,  ansible --version

**Comment by RNarmatha** [comment]
Could you please run this command outside of the cqube directory and share the screenshot

**Reply by vskmizoram** [reply]
![WhatsApp Image 2024-01-22 at 15 47 17_3915c26c](https://github.com/Sunbird-cQube/community/assets/157444637/7bb6fe5f-b869-4813-87ee-43c2086d040c)

**Comment by vskmizoram** [comment]
Yes its version is same even after running outside the directory

**Reply by RNarmatha** [reply]
@bommalla- please support this issue

**Comment by bommalla** [comment]
@vskmizoram @RNarmatha 

Please confirm me which version of cQube are you installed ?

**Reply by RNarmatha** [reply]
@bommalla- version 5.0.3

**Comment by bommalla** [comment]
@lokeshshivamurthy @vskmizoram @RNarmatha 

Actually your using ansible 2.10.8 old version but latest version is ansible 2.15.5 please update to new version.please find the attached screenshot for your reference.
![image](https://github.com/Sunbird-cQube/community/assets/86966482/4c047b23-114c-4710-a71b-a786d53816e0)

**Reply by RNarmatha** [reply]
@lokeshshivamurthy , @vskmizoram - kindly update the status

**Comment by vskmizoram** [comment]
Its all done now.. no issue at present
